describe.py

```diff
@@ -356,7 +356,12 @@ def document_collection(resource, path, root_discovery, discovery, css=CSS):
 
 
 def document_collection_recursive(
-    resource, path, root_discovery, discovery, doc_destination_dir
+    resource,
+    path,
+    root_discovery,
+    discovery,
+    doc_destination_dir,
+    artifact_destination_dir=DISCOVERY_DOC_DIR,
```

Comment:
Since this script is now accessible via the symlink from multiple directories, we have to ensure the `DISCOVER_DOC_DIR` default is set correctly above (line ~40). Shouldn't that line be `DISCOVERY_DOC_DIR = (pathlib.Path(__file__).resolve().parent / …)` rather than `...parent.resolve()…` as we have now?

IIUC, the way the variable is defined now makes sense for the script being at the repo top level, but if it's being called as a symlink in the `scripts/` subdirectory, it's not clear `__file__` will resolve to the link target, so it seems we need to `resolve()` as early as possible so we know where we are.